policy/modules/system/init.te

```diff
@@ -23,12 +23,21 @@ gen_tunable(init_upstart, false)
 ## </desc>
 gen_tunable(init_daemons_use_tty, false)
 
-## <desc>
-## <p>
-## Enable systemd to mount on all non-security files.
-## </p>
-## </desc>
-gen_tunable(init_mounton_non_security, false)
+ifdef(`init_systemd',`
+	## <desc>
+	## <p>
+	## Enable systemd to mount on all non-security files.
+	## </p>
+	## </desc>
+	gen_tunable(init_mounton_non_security, true)
+',`
+	## <desc>
+	## <p>
+	## Enable systemd to mount on all non-security files.
+	## </p>
+	## </desc>
+	gen_tunable(init_mounton_non_security, false)
+')
```

Comment:
No, we're not having this. Distros can use booleans.conf to override defaults.

Reply:
Good point. I'll drop this change.